### PR TITLE
Amend readmes

### PR DESCRIPTION
**Pull request by JasonBellONS**
## Synopsis

Amendments to thousand_pounds Readme/example

## Checklist

- [x] Documentation created/updated
- [x] Tests created/updated

## Description

Add a more detailed description of the pr including any necessary background
information.

### CODE REVIEW

File: sml_small/editing/thousand_pounds/README.md [hunk 1]
```diff
@@ -68,24 +71,23 @@ output = thousand_pounds(
...
-|principal_identifier|principal_variable|predictive|aux|threshold_upper|threshold_lower|TPC_marker|ratio|principal_adjusted_value|target_variable|target_original_value|target_adjusted_value
-|---|---|---|---|---|---|---|---|---|---|---|---|
-12340000001-201409-q100|50000000|60000|15000|1350|350|C|1000.0|50000.0|q101|500|0.5
-12340000001-201409-q100|-|-|-|-|-|-|-|-|q102|1000|1
-12340000001-201409-q100|-|-|-|-|-|-|-|-|q103|1500|1.5
-12340000001-201409-q100|-|-|-|-|-|-|-|-|q104||
+| unique_identifier       |principal_variable|predictive|aux|threshold_upper|threshold_lower|TPC_marker|ratio|principal_adjusted_value|target_variable|target_original_value|target_adjusted_value
+|-------------------------|---|---|---|---|---|---|---|---|---|---|---|
+ 12340000001-201409-q100 |50000000|60000|15000|1350|350|C|1000.0|50000.0|q101|500|0.5
+ 12340000001-201409-q100 |-|-|-|-|-|-|-|-|q102|1000|1
```
Comment (jhmjam) [inline sml_small/editing/thousand_pounds/README.md:91]: should there be tpc values here?

**Review by jhmjam — CHANGES_REQUESTED** [review]

File: sml_small/editing/thousand_pounds/README.md (hunk 1 above)
Comment (JasonBellONS) [reply sml_small/editing/thousand_pounds/README.md:91]: tpc marker and ratio is in there, the table view makes a bit more sense all rows are a single test case, its just how the different target variables have been represented

Comment (JasonBellONS) [reply sml_small/editing/thousand_pounds/README.md]: table representation is a bit weird here, the entire table is a single test case, with each row representing the principal variable and each target variable, so tpc marker and ratio are there for the first row,

amended ratio name (should be tpc_ratio), also removed target_original_value as no longer in output

Comment (jhmjam) [reply sml_small/editing/thousand_pounds/README.md]: Ok happy with this will do a read through and approve it from there :)

**Review by jhmjam — APPROVED** [review]

**Pull request merged**